bin/citgm-all.js

```diff
@@ -56,7 +65,9 @@ var options = {
   level: app.verbose,
   npmLevel: app.npmLoglevel,
   timeoutLength: app.timeout,
-  tmpDir: app.tmpDir
+  tmpDir: app.tmpDir,
+  withTags: app.withTags,
+  excludeTags: app.excludeTags
```

Comment:
with regard to 1:
you can't use split there because the code will break if `withTags` or `excludeTags` is undefined.